plasticos_automation/models/automation_config.py

```
@@ -0,0 +1,104 @@
+        help="Default minimum stock level. Per-product overrides take precedence.",
+    )
+
+    active = fields.Boolean(default=True)
+
+    _sql_constraints = [
+        (
+            "singleton_check",
+            "CHECK(id IS NOT NULL)",
+            "Only one automation configuration record is allowed.",
+        ),
+    ]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)